pkg/admissioncontroller/webhook/admission/auditpolicy/handler_test.go

```diff
@@ -540,6 +541,22 @@ rules:
 				test(admissionv1.Create, nil, shootv1beta1K8sV124, false, statusCodeInvalid, "audit policy with apiVersion 'v1alpha1' is not supported for kubernetes version >= 1.24.0", "")
 			})
 
+			It("references a valid auditPolicy/v1alhpa1 (UPDATE kubernetes version)", func() {
```

Comment:
```suggestion
			It("references a valid auditPolicy/v1alpha1 (UPDATE kubernetes version)", func() {
```
and other occurrences as well.